File: dataset_reduzido/avatar/Python/Code/atcoder_ABC070_B.py
A = list ( map ( int , input ( ).split ( ) ) )
if A [ 1 ] <= A [ 2 ] or A [ 3 ] <= A [ 0 ] :
    print ( 0 )
else :
    A.sort ( )
    print ( A [ 2 ] - A [ 1 ] )

File: output/Llama-2-100/avatar/Python/Java/atcoder_ABC070_B.java
List<Integer> A = Arrays.asList ( input ( ).split ( ) .map ( Integer::new ) .toArray ( ) );
if ( A.get ( 1 ) <= A.get ( 2 ) or A.get ( 3 ) <= A.get ( 0 ) ) :
    System.out.println ( 0 );
else :
    A.sort ( );
    System.out.println ( A.get ( 2 ) - A.get ( 1 ) );


